yarn.lock

```diff
@@ -16033,7 +16033,17 @@ create-hash@^1.1.0, create-hash@^1.1.2, create-hash@^1.2.0:
     ripemd160 "^2.0.1"
     sha.js "^2.4.0"
 
-create-hmac@^1.1.4, create-hmac@^1.1.7:
+create-hash@~1.1.3:
```

Comment:
That's what we call a feature-complete library - it was perfect 8 years ago, and it's still perfect 😄 

![image](https://github.com/user-attachments/assets/cd7dfdf4-0cf9-4842-b1b7-785bfbb48e8f)